### PR TITLE
Category colours based on theme colours

### PR DESCRIPTION
**Pull request by Larspolo**
## Description

First 9 are now set, after that it is random generated based on lighness levels between 0.2 and 0.5 (was 0.4 and 0.8, but I ofound that too bright)
image

### CODE REVIEW

File: src/vue/src/utils/colors.js [hunk 1]
```diff
@@ -88,8 +88,24 @@ export default {
     },
 
     randomBrightRGBcolor () {
-        const hsl = this.randomBrightHSLColor()
+        const hsl = this.randomBrightHSLcolor()
         const rgb = this.hslToRgb(hsl.hue * 360, hsl.saturation, hsl.lightness)
         return this.rgbToHex(rgb.red, rgb.green, rgb.blue)
     },
+
+    themeColors: [
```
Comment (mjvkeulen) [inline src/vue/src/utils/colors.js:96]: This should probably come from a source that is also accessible by the backend, so we can make use of this in factory/category.py and presetdb

Comment (Larspolo) [reply src/vue/src/utils/colors.js:96]: Fine to move this to an issue?

**Comment by Larspolo** [comment]
image

Cross is barely to not visible for red (ish) coloured categories

**Comment by Larspolo** [comment]
image
Categories without description have a different size

File: src/vue/src/utils/colors.js (hunk 1 above)
Comment (engelhamer) [reply src/vue/src/utils/colors.js]: Now using the sass colormap of `themeColors` (which had the same name, so was quite confusing anyways) without the gray shades.

Comment (mjvkeulen) [reply src/vue/src/utils/colors.js]: Moved my remark to Z#1542

**Review by mjvkeulen — APPROVED** [review]

**Pull request merged**